projects/plugins/jetpack/_inc/lib/class.media-extractor.php

```diff
@@ -543,14 +549,21 @@ public static function get_images_from_html( $html, $images_already_extracted, $
 			}
 
 			if ( ! in_array( $queryless, $image_list, true ) ) {
-				if ( $extract_alt_text && ! empty( $extracted_image['alt_text'] ) ) {
-					$image_list[] = array(
-						'url'      => $queryless,
-						'alt_text' => $extracted_image['alt_text'],
-					);
+				$image_to_add = array(
+					'url' => $queryless,
+				);
+				if ( $extract_alt_text ) {
+					if ( ! empty( $extracted_image['alt_text'] ) ) {
+						$image_to_add['alt_text'] = $extracted_image['alt_text'];
+					}
+					if ( ! empty( $extracted_image['src_width'] ) || ! empty( $extracted_image['src_width'] ) ) {
```

Comment:
I think the second check should be on `src_height` and not `src_width` again.

Reply:
Thanks. I fixed it!